raft/src/main/java/org/apache/kafka/raft/RaftConfig.java

```diff
@@ -129,16 +133,41 @@ public String toString() {
                 QUORUM_LINGER_MS_DOC);
     }
 
+    private final int requestTimeoutMs;
+    private final int retryBackoffMs;
+    private final int electionTimeoutMs;
+    private final int electionBackoffMaxMs;
+    private final int fetchTimeoutMs;
+    private final int appendLingerMs;
+
     public RaftConfig(Properties props) {
```

Comment:
nit: I think this constructor is redundant. can we remove it?

Reply:
Looks like there are a bunch of tests that seem to be using this constructor
- `testSingleQuorumVoterConnections`
- `testMultiQuorumVoterConnections`
- `testInvalidQuorumVotersConfig`